test/kube_testing/k8s.go

```diff
@@ -314,27 +312,42 @@ func (o *K8s) initNamespace() {
 // Delete POD with completion check
 // Make force delete on timeout
 func (o *K8s) deletePods(pods ...*v1.Pod) error {
-	var ctx []context.Context
-	for _, pod := range pods {
-		delOpt := &metaV1.DeleteOptions{}
-		err := o.clientset.CoreV1().Pods(pod.Namespace).Delete(pod.Name, delOpt)
-		if err != nil {
-			return err
-		}
-
-		c, cancel := context.WithTimeout(context.Background(), podDeleteTimeout)
-		ctx = append(ctx, c)
-		defer cancel()
-	}
+	var wg sync.WaitGroup
 
-	for i, pod := range pods {
-		err := blockUntilPodWorking(o.clientset, ctx[i], pod)
-		if err != nil {
-			err = o.deletePodForce(pod)
-			logrus.Warnf(`The POD "%s" may continue to run on the cluster`, pod.Name)
+	for _, my_pod := range pods {
+		wg.Add(1)
+		pod := my_pod
+		go func() {
```

Comment:
OK, taht is interesting. Is it faster to run these as threads?

Reply:
A bit faster, but just a bit.